reflex/el/elements/base.py

```diff
@@ -0,0 +1,60 @@
+"""Element classes. This is an auto-generated file. Do not edit. See ../generate.py."""
+from typing import Union
+
+from reflex.el.element import Element
+from reflex.vars import Var as Var_
+
+
+class BaseHTML(Element):
+    """Base class for common attributes."""
+
+    #  Provides a hint for generating a keyboard shortcut for the current element.
+    access_key: Var_[Union[str, int, bool]]
```

Comment:
i'd like to read this type as `access_key: Var_[Attribute]` or maybe `Var_[HTMLAttribute]`.

something that reduces the complexity of what my brain has to grok to understand the line.